<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iNaturalist Observations</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>431</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load observations</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="filters">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Filters</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Taxon name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_species">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>431</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_from">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>date from:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_country">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Country:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_countries">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>431</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_date_from">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="specialValueText">
    <string>None</string>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>1900</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_date_range">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_to">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>date to:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_date_to">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2200</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_map_extent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Map extent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_species</tabstop>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>comboBox_countries</tabstop>
  <tabstop>checkBox_date_range</tabstop>
  <tabstop>dateEdit_date_from</tabstop>
  <tabstop>dateEdit_date_to</tabstop>
  <tabstop>checkBox_map_extent</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_date_range</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateEdit_date_from</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_date_range</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateEdit_date_to</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
